x-pack/plugins/maps/public/index.ts

```diff
@@ -5,10 +5,16 @@
  */
 
 import { PluginInitializer } from 'kibana/public';
+import { PluginInitializerContext } from 'kibana/public';
 import { MapsPlugin, MapsPluginSetup, MapsPluginStart } from './plugin';
+import { MapsXPackConfig } from '../config';
 
-export const plugin: PluginInitializer<MapsPluginSetup, MapsPluginStart> = () => {
-  return new MapsPlugin();
+// @ts-ignore
```

Comment:
Can you remove `ts-ignore` from this file. Just tested locally, and with recent changes, they are not needed.